packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -127,37 +125,23 @@ function useMovingAnimation(
 		reset: triggeredAnimation !== finishedAnimation,
 		config: { mass: 5, tension: 2000, friction: 200 },
 		immediate: prefersReducedMotion,
-		onFrame: ( props ) => {
+		onFrame: ( { x, y } ) => {
 			if (
 				adjustScrolling &&
 				scrollContainer.current &&
 				! prefersReducedMotion &&
-				props.y
+				y
 			) {
-				scrollContainer.current.scrollTop =
-					transform.scrollTop + props.y;
+				scrollContainer.current.scrollTop = transform.scrollTop + y;
 			}
+
+			ref.current.style.transformOrigin = 'center';
+			ref.current.style.transform =
+				x === 0 && y === 0 ? null : `translate3d(${ x }px,${ y }px,0)`;
+			ref.current.style.zIndex =
+				! isSelected || ( x === 0 && y === 0 ) ? null : '1';
```

Comment:
hah :) so you're recreating react-spring. It would be interesting to see how this differs from React spring internally. I mean shouldn't we be able to pass a ref for react-spring to do that.

Reply:
Ok, just read the use-spring issue. i wonder what's different between this and `applyAnimatedValues` calls? Also should we upgrade?

Reply:
Not really as far as recreating. It still provides the values, we just take render it here instead of passing instructions on how to render it for us. Ideally it would be great if we could just pass the ref and instructions to `useSpring`, but there's no big difference.

Reply:
yes, so it looks like this is the function that runs in the library https://github.com/react-spring/react-spring/blob/master/src/targets/web/globals.ts#L82-L127 which is pretty similar to what you did for the "styles" here.

Reply:
While testing though, it seems  the animation is broken for me (blocks overlap)

Reply:
Blocks overlapping sounds like the styles are not being set. It works fine for me, but I'll test again.

Exactly, we're just setting the styles directly ourselves instead of letting `react-spring` doing it on the element they rendered.

There's a v9 is progress but it hasn't been released yet.

Reply:
Strange, the blocks are animating the same as master for me in FF, Chrome and Safari.

Reply:
I tested on Arch Linux using a browser based on Chromium 80, and the block movement animations all seem fine to me. I also tested in Firefox Developer Edition 78.0b2 and everything worked there as well.

Reply:
That's weird, they overlap for me regardless of the browser (Chrome and Safari)

Reply:
@youknowriad Did you do a full refresh etc? It works fine for me. I also tried switching branches a few times, but I can't reproduce any overlap issues

Reply:
So I just tested for the third time and this time it worked. Sorry for delaying that.

Reply:
No worries :) I'm glad it works now